src/ZfBugsnag/Service/BugsnagService.php

```diff
@@ -67,33 +67,49 @@ public function setupErrorHandlers()
      */
     public function errorHandler($errno, $errstr, $errfile = '', $errline = 0, $errcontext = [])
     {
-        $result = $this->bugsnag->errorHandler($errno, $errstr, $errfile, $errline);
+        try
+        {
+            $this->bugsnag->errorHandler($errno, $errstr, $errfile, $errline);
+        } catch (\Exception $e) {
+            // Something wrong in the bugsnag notify call
```

Comment:
I'm hesitant to add too much code here since we're already in an error state.  I don't want the call to ZendLog to end up hitting the error/exception handlers themselves, so we'd have to wrap those in a try/catch as well, which I think puts us in the same boat (what do we do in the catch of that call?).

Maybe it makes sense to provide a config option for the module that is something that is callable that we can call in the event that bugsnag notification fails here. Then the user can decide how to handle the error condition (user could error_log here, or trigger_error, or open a stream, etc...).

Another possible option could be to add some information to the error that is going to get logged (based on user's php config).  As an example, we could add something like "Additionally the Bugsnag notification failed for this error (bugsnag exception message)".

I looked around at some other modules that set an error or exception handler, but 1) there aren't many and 2) they don't really deal with this condition.

I'll keep looking around and testing though.